imapserver/status.go

```diff
@@ -86,6 +86,9 @@ func (c *Conn) writeStatus(data *imap.StatusData, options *imap.StatusOptions) e
 	if options.NumRecent {
 		listEnc.Item().Atom("RECENT").SP().Number(*data.NumRecent)
 	}
+	if options.HighestModSeq {
+		listEnc.Item().Atom("HIGHESTMODSEQ").SP().Number64(int64(data.HighestModSeq))
```

Comment:
We should be able to use `ModSeq` instead of `Number64` here. This would avoid sending a negative number if a modseq is high.